src/index.html

```diff
@@ -9,10 +9,72 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
It seems like you're trying to link a SCSS file directly in your HTML. Browsers don't understand SCSS files natively. You should compile your SCSS to CSS and link the resulting CSS file instead.